engine/src/main/scala/cromwell/engine/package.scala

```diff
@@ -43,6 +42,10 @@ package object engine {
   class CromwellFatalException(exception: Throwable) extends Exception(exception)
   class PreemptedException(msg: String) extends Exception(msg)
 
+  object CromwellConfiguration {
+    val config: Config = ConfigFactory.load
```

Comment:
I don't feel that this extra layer of singleton adds any benefit, since `ConfigFactory.load` is already [cached](http://typesafehub.github.io/config/v1.3.0/com/typesafe/config/ConfigFactory.html#load--).

Why don't callers who like using singletons (instead of passing in configurations) just use the shorter `ConfigFactory.load` instead of `CromwellConfiguration.Config`?